packages/react-components/src/components/VideoGallery/styles/VideoGalleryResponsiveVerticalGallery.styles.ts

```diff
@@ -13,24 +14,36 @@ import { LARGE_FLOATING_MODAL_SIZE_PX, SMALL_FLOATING_MODAL_SIZE_PX } from './Fl
  *
  * @private
  */
-export const LARGE_VERTICAL_GALLERY_TILE_SIZE_REM = { minHeight: 5.625, maxHeight: 9, width: 9 };
+export const VERTICAL_GALLERY_TILE_SIZE_REM = { minHeight: 5.625, maxHeight: 9, width: 9 };
 
 /**
  * Styles for the VerticalGallery's container set in parent.
  *
  * width is being increased by 1rem to account for the gap width desired for the VerticalGallery.
  *
  * @param shouldFloatLocalVideo whether rendered in floating layout or not
- * @param isNarrow is being rendered with narrow tile settings.
  * @returns Style set for VerticalGallery container.
  */
-export const verticalGalleryContainerStyle = (shouldFloatLocalVideo: boolean, isNarrow: boolean): IStyle => {
+export const verticalGalleryContainerStyle = (shouldFloatLocalVideo: boolean): IStyle => {
   return {
-    width: `${LARGE_VERTICAL_GALLERY_TILE_SIZE_REM.width + 1}rem`,
-    height: shouldFloatLocalVideo
-      ? isNarrow
-        ? `calc(100% - ${_pxToRem(SMALL_FLOATING_MODAL_SIZE_PX.height)})`
-        : `calc(100% - ${_pxToRem(LARGE_FLOATING_MODAL_SIZE_PX.height)})`
-      : '100%'
+    width: `${VERTICAL_GALLERY_TILE_SIZE_REM.width}rem`,
+    height: shouldFloatLocalVideo ? `calc(100% - ${_pxToRem(VERTICAL_GALLERY_FLOATING_MODAL_SIZE_PX.height)})` : '100%',
+    paddingBottom: '0.5rem'
   };
 };
+
+/**
+ * @private
+ */
+export const VERTICAL_GALLERY_TILE_STYLE = {
+  minHeight: `${VERTICAL_GALLERY_TILE_SIZE_REM.minHeight}rem`,
+  minWidth: `${VERTICAL_GALLERY_TILE_SIZE_REM.width}rem`,
+  maxHeight: `${VERTICAL_GALLERY_TILE_SIZE_REM.maxHeight}rem`,
+  maxWidth: `${VERTICAL_GALLERY_TILE_SIZE_REM.width}rem`
```

Comment:
I think we can just define this as width and we don't need a min and max here

Reply:
I remember there was an issue with that simplification from the past but let me test if I can make that simplification.